.github/workflows/electron-linux.yml

```diff
@@ -0,0 +1,47 @@
+name: Electron Rebuild CI - Linux (x32 & x64)
+
+on: [push, pull_request]
+
+jobs:
+  build:
+
+    runs-on: ubuntu-latest
+
+    strategy:
+      matrix:
+        electron:
+          - 5
+          - 6
+          - 7
+          - 8
+        include:
+          # includes a new variable for each electron (essentially a switch statement) 
+          - electron: 5
+            nodeVersion: 70
+          - electron: 6
+            nodeVersion: 73
+          - electron: 7
+            nodeVersion: 75
+          - electron: 8
+            nodeVersion: 76
+
+    steps:
+    - uses: actions/checkout@v2
+    - name: Use Node.js
+      uses: actions/setup-node@v1
+      with:
+        os: ${{ matrix.os }}
+    - name: Install packages
+      run: npm install --unsafe-perm
+    - name: Install electron-rebuild
+      run: npm install --save-dev electron-rebuild@latest
+    - name: Run on ${{ matrix.os }} - Electron v${{ matrix.electron }} (Node ${{ matrix.nodeVersion }})
+      run: ./node_modules/.bin/electron-rebuild -v=${{ matrix.electron }} -w node-sass
+    - uses: actions/upload-artifact@v1
+      with:
+        name: linux_x64_${{ matrix.nodeVersion }}_binding.node
+        path: build/Release/binding.node
+    # npm test failing on pre-existing master issues, commenting out
```

Comment:
What the pre-existing failures?

Reply:
If you see this run
https://github.com/glenn2223/node-sass/runs/514109989?check_suite_focus=true#step:9:11601

Reply:
OK, the other issue I could think that it's running into is that the `npm test` is still using the regular system NodeJS version to execute, but the binding is setup for an electron environment. Not familiar with how to properly test Electron, but is there something like `export NODE=electron_binary`, similar to what python tests do with `export PYTHON=python3` for making sure a particular version is used.

If it really is a CLI issue with Electron, but Electron won't use the test, maybe there is a way to just skip the test on Electron, but run the rest.

Reply:
> I'll add the data from a local run tomorrow (in GMT here) as I think the error was different on that

```console
  6084 passing (31s)
  753 pending
  2 failing

  1) cli node-sass sass/ --output css/ should not error if output directory is a symlink:
     Error: EEXIST: file already exists, mkdir 'C:\Users\Glenn\Documents\GitHub\node-sass\test\fixtures\input-directory\css'
      at Object.mkdirSync (fs.js:860:3)
      at Context.<anonymous> (test\cli.js:582:10)
      at processImmediate (internal/timers.js:456:21)

  2) cli node-sass --follow --output output-dir input-dir should compile with the --follow option:
     Error: EEXIST: file already exists, mkdir 'C:\Users\Glenn\Documents\GitHub\node-sass\test\fixtures\follow\input-dir'
      at Object.mkdirSync (fs.js:860:3)
      at Context.<anonymous> (test\cli.js:623:10)
      at processImmediate (internal/timers.js:456:21)



npm ERR! Test failed.  See above for more details.
```

**The above is the result when I run `npm test` on the original repository (no changes)**  
Have it open in VSCode Winx62, using vendor node from win32-x64-79 (I have Node 13.10.1)

Reply:
> OK, the other issue I could think that it's running into is that the `npm test` is still using the regular system NodeJS version to execute, but the binding is setup for an electron environment. Not familiar with how to properly test Electron, but is there something like `export NODE=electron_binary`, similar to what python tests do with `export PYTHON=python3` for making sure a particular version is used.
> 
> If it really is a CLI issue with Electron, but Electron won't use the test, maybe there is a way to just skip the test on Electron, but run the rest.

TBH, I don't know why I'm running `npm test` as we can't test it in that environment (unless I missed something)

The test is running on the system Node but still using the binary it downloaded from GitHub to test. This is because running `electron-rebuild` places the electron binary in the `bin` & `build/Release` folders, completely separate from where the current binary is (obv).

I tried (even though I knew the result) to replace the binary in the `vendor` folder and see what would happen; you guessed it, I got the below error: (essentially trying to run Electron 8 on Node 13)
```console err
Binary has a problem: Error: The module '\\?\C:\Users\Glenn\Documents\GitHub\node-sass\vendor\win32-x64-79\binding.node'
was compiled against a different Node.js version using
NODE_MODULE_VERSION 76. This version of Node.js requires
NODE_MODULE_VERSION 79. Please try re-compiling or re-installing
the module (for instance, using `npm rebuild` or `npm install`).
```

I think we should drop the `npm test` as the binary it is rebuilding off must be functional otherwise it wouldn't be accessible in a public release (or at least that's what we all hope 😅)

Do you agree?